grub/02_ignition_firstboot

```diff
@@ -9,5 +9,16 @@ search --set=bootpart --label boot
 # to be used later on the kernel command line.
 set ignition_firstboot=""
 if [ -f "(${bootpart})/ignition.firstboot" ]; then
-    set ignition_firstboot="ignition.firstboot rd.neednet=1 ip=dhcp"
+    # default to dhcp networking parameters to be used with ignition 
+    ignition_network_kcmdline='rd.neednet=1 ip=dhcp'
```

Comment:
Should we have `rd.neednet=1` in the default set and only be overriding the `ip=dhcp`?

I could see a future where the installer is running without needing internet access in the initramfs (for instance it could be eventually updated to allow for the install of a local image / ignition config).

Reply:
this change won't have any effect on the installer itself AFAIU - those options are controlled by either the admin setting up the PXE config or [1](https://github.com/coreos/fedora-coreos-config/blob/d4ce30fad26d9e5f944c55e44446b5930255e766/installer/isolinux/isolinux.cfg#L68) [2](https://github.com/coreos/fedora-coreos-config/blob/d4ce30fad26d9e5f944c55e44446b5930255e766/installer/EFI/fedora/grub.cfg#L31) for the ISO image.

Reply:
Correct, this won't affect the installer. However the scenario I posted allows for a situation where the installer does not require networking inside of the initramfs (meaning it can run successfully without `rd.neednet=1`) but Ignition would require networking to perform it's actions.

Reply:
I think I see what you're saying. Basically installer boot doesn't require networking at all but ignition first boot does. I'd say we'd have to force the user to use the needed network config during the installer run (or stop the first boot and add kernel args there). 

This is hypothetical right now I think because we don't actually provide any media that contains the dd image embedded in it so maybe we cross that bridge when we get there.